src/app/globals.css

```diff
@@ -100,43 +109,47 @@ form label {
 }
 
 #my-gui-container {
-  width: 200px;
-  position: fixed;
-  top: 100px;
-  right: 10px;
+  width: 13rem;
+  position: absolute;
```

Comment:
hmm if we are making the gui positioning absolute (which i think is fair) then we should make it so only the left info container is scrollable